about.html

```diff
@@ -0,0 +1,135 @@
+          <div></div>
+          <div></div>
+          <div></div>
+      </div>
+
+      <!-- Desktop Navigation -->
+      <nav class="desktop-nav-1">
+          <ul class="flex">
+              <li><a href="$"><img class="facebook" src="./cap-detials/facebook.png" alt="facebook"></a></li>
+              <li><a href="$"><img class="twitter" src="./cap-detials/twitter.png" alt="twitter"></a></li>
+              <li><a href="#">English</a></li>
+              <li><a href="#">My Page</a></li>
+          </ul>
+      </nav>
+
+      <nav class="desktop-nav-2 flex">
+          <a href="./index.html"><img src="./cap-detials/fogo.png" alt="Logo"></a>
+          <ul class="flex">
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#program">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#nominees">Nominees</a></li>
+              <li><a href="#partners">Partners</a></li>
+              <li><button type="button">Vote Now</button></li>
+          </ul>
+          
+      </nav>
+  </header>
+
+
+
+  <main>
+    <section class="about-intro flex">
+      <div class="intro-heading flex about-intro-heading">
+        <h2>"<h2>"Hello there! Footballl Lovers"</h2>
+        <h1>Ziggies <br>  Award Ceremony 2022</h1>
+        <hr>
+      </div>
+      <p class="mobile-paragraph about-mobile-paragraph">Ziggies Award ceremony is an award ceremony to crown the best player for the 2022 season which takes place in France.
+        our Award Ceremony brings together top coaches,Players & journalists from all over the world for a unique and joyful experience.</p>
+      <p class="about-Email-paragraph">Please contact us per Email for any further questions about EDM Global Summit!
+      </p>
+      <a class="about-Email-paragraph about-Email" href="mailto:contactus@ziggiesawardshow.org">contactus@ziggiesawardshow.org</a>
+    </section>
+  </main>
+
+  <section class="edmLogo flex">
+    <h2>Ziggies award show  2022 Logo</h2>
+    <hr>
+    <p class="about-Email-paragraph">The Logo of Ziggies award show  was designed by mark Otuya.</p>
+    <div class="aboutEdmLogo flex">
+      <img src="./cap-detials/fogo.png" alt="Ziggies award show  Logo">
+    </div>
+  </section>
+
+  <section class="edmLogo flex">
+    <h2>See the past Ziggies award show </h2>
+    <hr>
+    <p class="about-Email-paragraph">Take a look at past two Ziggies award shows which took place in london and Germany.
+    </p>
+    <div class="desktop-summit flex">
+      <div class="summit1">
+        <div class="summit-overlay"></div>
+        <section class="Summit-text">
+          <h2>2021</h2>
+          <span class="flex">Ziggies award show 2021 in london</span>
+        </section>
+      </div>
+
+      <div class="summit2">
+        <div class="summit-overlay"></div>
+        <section class="Summit-text">
+          <h2>2020</h2>
+          <span class="flex">Ziggies award show 2021 in Germany</span>
+        </section>
```

Comment:
- [ ] Great job so far 👏🏻  However you are supposed to follow the provided design guidelines for this project, Kindly improve your UI design, you could add the background overlay color 👍 

- Your Design:

![29rrrr](https://user-images.githubusercontent.com/79058364/176434702-c18b1119-b7f7-4880-b62b-e725189684ca.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![Annotation 2022-05-18 110524](https://user-images.githubusercontent.com/85113567/169014609-2f09ccf3-cab1-4dda-b381-8e449c302275.jpg)